ui/tests/integration/components/path-filter-config-list-test.js

```diff
@@ -41,7 +40,7 @@ module('Integration | Component | path filter config list', function (hooks) {
 
   hooks.beforeEach(function () {
     this.context = { owner: this.engine }; // this.engine set by setupEngine
-    const ajaxStub = sinon.stub().usingPromise(Promise);
+    const ajaxStub = sinon.stub();
```

Comment:
Interesting is this method deprecated? I've always used `stub.resolves` or `stub.rejects` so guessing this is an old pattern or otherwise unnecessary.

Reply:
just the `.usingPromise` method I believe.
>In recent versions, Sinon has standardized on using the native Promise implementation and has deprecated the usingPromise method. This change means that attempting to use usingPromise will result in the error you're seeing.

Reply:
You should still be able to use r[esolve and reject](https://sinonjs.org/releases/latest/promises/)